<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataPointmap_PointsManager_EditPointsDialog</class>
 <widget class="QDialog" name="GeoDataPointmap_PointsManager_EditPointsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Points</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Properties of selected points</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numPointsLabel">
        <property name="text">
         <string>Number of Points:</string>
        </property>
        <property name="buddy">
         <cstring>numPointsValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="numPointsValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="minLabel">
        <property name="text">
         <string>Minimum value:</string>
        </property>
        <property name="buddy">
         <cstring>minValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="RealNumberEditWidget" name="minValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="maxLabel">
        <property name="text">
         <string>Maximum value:</string>
        </property>
        <property name="buddy">
         <cstring>maxValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="RealNumberEditWidget" name="maxValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="xLocLabel">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="RealNumberEditWidget" name="xEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="yLocLabel">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="RealNumberEditWidget" name="yEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="RealNumberEditWidget" name="valueEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataPointmap_PointsManager_EditPointsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataPointmap_PointsManager_EditPointsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
